### ISSUE TITLE
Question about the training time reported in the paper.

### ISSUE THREAD

**Issue by wangm-buaa** [AIBluefisher/DOGS#9]
Thank you for releasing such an excellent open-source work.
I have a question regarding the training time reported in the paper. How is this time calculated? Is it the average computation time per block, the computation time of the slowest block, or the total computation time across all blocks?
![image](https://github.com/user-attachments/assets/df576bf1-7b4a-4791-868d-17681141dd20)

**Comment by AIBluefisher** [comment]
Hi, we record the training time on the master node, which starts from loading the dataset and ends when all blocks finish the training.

**Comment by wangm-buaa** [comment]
Thank you for your response. I have another question: I noticed that the code seems to involve a 'color_correct' operation. Were the data provided in the paper obtained after applying this processing?

![image](https://github.com/user-attachments/assets/65a7f9f3-faa3-4942-ac74-b1c72363c482)

https://github.com/AIBluefisher/DOGS/blob/ffde6e8dc3846f1dafec1e2e37faf208448bd794/conerf/evaluators/gaussian_splatting_evaluator.py#L292C9-L312C70

**Comment by AIBluefisher** [comment]
Yes. We followed VastGaussian to use the color correction during evaluation.
